sdk/core/azure-core/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 - [[#4213]](https://github.com/Azure/azure-sdk-for-cpp/issues/4213) Fixed a bug where `Host` request header is not set for non-default port (80, 443).
 - [[#4443]](https://github.com/Azure/azure-sdk-for-cpp/issues/4443) Fixed potentially high CPU usage on Windows.
+- [[#4490]](https://github.com/Azure/azure-sdk-for-cpp/issues/4490) Fixed WinHTTP memory leak during failed requests.
```

Comment:
I am surprised you don't have merge conflicts here.
Please update your local sources. The change will go to the entries for the next release (be it `1.8.2`, `1.9.0`, or `1.9.0-beta.2`)